en/includes/guides/organization-management/share-user-profiles.md

```diff
@@ -0,0 +1,45 @@
+# Share user profiles with organizations
+
+If your organization users need access to organizations lower in the hierarchy without creating accounts for each and every organization, you can simply share these user profiles with those organizations. This is ideal when a user needs access to multiple organizations while maintaining a single identity. It also reduces the overhead of managing the user's profile across multiple organizations.
+
+## Mathods to share profiles
+
+There are three ways how user profiles are shared between organizations:
+
+1. When a root organization user creates one or more organizations, the user automatically becomes the admin of these organizations and the user profile is shared with them. 
```

Comment:
We can't say automatically become an admin of the created org.

If the root org user creates the organization via console, yes become an admin of created org with getting Console Administartor role.
If org creation is done through API calls/ another app, organization role assignment needs to be done manually